site/content/en/docs/manual/advanced/formats/_index.md

```diff
@@ -1,28 +1,103 @@
+| [CamVid 1.0](format-camvid)                                                                                                         | .txt <br>.png | Semantic <br>Segmentation                                   | U-Net, SegNet, DeepLab, <br>PSPNet, FCN, Mask R-CNN, <br>ICNet, ERFNet, HRNet, <br>V-Net, and others.                                                                                   | Bounding Boxes, Polygons                                                               | Not supported        | Not supported |
+| [Cityscapes 1.0](format-cityscapes)                                                                                                 | .txt<br>.png  | Semantic<br>Segmentation                                    | U-Net, SegNet, DeepLab, <br>PSPNet, FCN, ERFNet, <br>ICNet, Mask R-CNN, HRNet, <br>ENet, and others.                                                                                    | Polygons, Bounding Boxes.                                                              | Specific attributes  | Not supported |
+| [COCO 1.0](format-coco)                                                                                                             | JSON          | Detection, Semantic <br>Segmentation                        | YOLO (You Only Look Once), <br>Faster R-CNN, Mask R-CNN, SSD (Single Shot MultiBox Detector), <br> RetinaNet, EfficientDet, UNet, <br>DeepLabv3+, CenterNet, Cascade R-CNN, and others. | Bounding Boxes, Polygons                                                               | Specific attributes  | Not supported |
+| [COCO Keypoings 1.0](coco-keypoints)                                                                                                | .xml          | Keypoints                                                   | OpenPose, PoseNet, AlphaPose, <br> SPM (Single Person Model), <br>Mask R-CNN with Keypoint Detection:, and others.                                                                      | Skeletons                                                                              | Specific attributes  | Not supported |
+| [CVAT for images 1.1](<[format-cvat](http://localhost:1313/docs/manual/advanced/formats/format-cvat/#cvat-for-videos-export)>)      | .xml          | Universal format<br> for all types of <br>annotations.      | Universal format<br> for all types of <br>models.                                                                                                                                       | Bounding Boxes, Polygons, <br>Polylines, Points, Cuboids, <br>Skeletons, Tags, Tracks. | All attributes       | Supported     |
```

Comment:
The link doesn't work. CVAT for images 1.1 doesn't support tracks. It is possible to export them, but import will be impossible.